docs/kubernetes/ingress-usage.md

```diff
@@ -736,6 +736,42 @@ spec:
           servicePort: 80
 ```
 
+#### Path ratelimit
+
+To ratelimit a specific path use a second ingress definition like
+
+```yaml
+apiVersion: extensions/v1beta1
```

Comment:
maybe for a dedicated PR, but the newest version is `networking.k8s.io/v1` since a few Kubernetes versions ago, so it may make sense to start documenting just that.

Reply:
Lets tackle that within https://github.com/zalando/skipper/issues/1824